02-notebooks-as-jats.md

```diff
@@ -6,6 +6,15 @@ date: 2022-12-09
 
 The goal of this article is to recommend community standards for representing Notebooks as [JATS](https://jats.nlm.nih.gov). As discussed in [approach](./approach.md), the notebook can either be the main document of a JATS `article` or as a `sub-article`, which is more appropriate for authors who create figures in different notebooks today, and reference the figures created in those notebooks in the main document.
 
+We propose that all referenced notebooks be represented as:
+
+- A rendering of the notebook, including source code, as a JATS `sub-article` of the main the JATS article.
```

Comment:
```suggestion
- A rendering of the notebook, including code and markdown cells, as a JATS `sub-article` of the main the JATS article.
```

Reply:
I think this is slightly clearer that the "source code" libraries are not included in the JATS, but the "source code" of the code cells is.